backend/database_handler/chain_snapshot.py

```diff
@@ -18,6 +18,7 @@ def __init__(self, session: Session):
         self.all_validators = self.validators_registry.get_all_validators()
         self.pending_transactions = self._load_pending_transactions()
         self.num_validators = len(self.all_validators)
+        self.accepted_transaction = self._load_accepted_transactions()
```

Comment:
```suggestion
        self.accepted_transactions = self._load_accepted_transactions()
```

Reply:
Solved in eb5ea52a2d4433e9b5e2441d05bf6f027712d486